docs/tutorials/pytorch-mnist-local-qs.rst

```diff
@@ -0,0 +1,114 @@
+.. _pytorch_mnist_quickstart:
+
+#########################################
+ Run Your First Experiment in Determined
+#########################################
+
+.. meta::
+   :description: Learn how to integrate the PyTorch MNIST model into Determined AI using only a single CPU or GPU.
+   :keywords: PyTorch API,MNIST,model developer,quickstart
+
+In this tutorial, we’ll show you how to integrate a training example with the Determined
+environment. We'll run our experiment on a local training environment requiring only a single CPU or
+GPU.
+
+.. note::
+
+   This tutorial is recommended as an introduction for model developers who are new to Determined
+   AI.
+
+**Objective**
+
+Our goal is to integrate the `PyTorch MNIST training example
+<https://github.com/pytorch/examples/blob/main/mnist/main.py>`_ into Determined in four steps:
+
+-  Download and extract the files
+-  Set up our training environment
+-  Run the experiment
+-  View the experiment in our browser
+
+**Prerequisites**
+
+-  `System Requirements
+   <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#system-requirements>`_
+-  `Docker
+   <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#install-docker>`_
+
+********************
+ Download the Files
+********************
+
+To get started, we'll first download and extract the files we need and ``cd`` into the directory.
+
+-  Download the :download:`mnist_pytorch.tgz </examples/mnist_pytorch.tgz>` file.
+-  Open a terminal window, extract the files, and ``cd`` into the ``mnist_pytorch`` directory:
+
+.. code::
+
+   tar xzvf mnist_pytorch.tgz
+   cd mnist_pytorch
+
+**********************************
+ Set Up Your Training Environment
+**********************************
+
+To start your experiment, you'll need a Determined cluster. If you are new to Determined AI
+(Determined), you can install the Determined library and start a cluster locally:
+
+.. code::
+
+   # If your machine has GPUs:
+   det deploy local cluster-up
+
+   # If your machine does not have GPUs:
+   det deploy local cluster-up --no-gpu
+
+.. note::
+
+   If you want to see if Determined is already installed, you can type ``det --version``.
+
+********************
+ Run the Experiment
+********************
+
+To run the experiment, enter the following command:
+
+.. code::
+
+   det experiment create const.yaml .
+
+A notification displays letting you know the experiment has started.
+
+.. code::
+
+   Preparing files (.../mnist_pytorch) to send to master...
+   Created experiment xxx
+
+.. note::
+
+   The Determined CLI may display a warning letting you know the requested job requires more slots
+   than currently available. For the purposes of this tutorial, you can ignore this warning.
```

Comment:
I'm not sure if this note is needed.  Either the user already has other jobs running, in which case they're not totally new, or they might be running on a cloud installation, in which case they're not actually following this quick start.

Reply:
removed